catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-forms.less

```diff
@@ -0,0 +1,36 @@
+@{customElementNamespace}search-form {
```

Comment:
We should rename this file to be `search-form.less`.

Reply:
Done